src/main/java/frc/robot/Constants.java

```diff
@@ -34,6 +34,12 @@ public enum ChassisConfiguration {
     PRACTICE
   }
 
+  public static ChassisConfiguration getChassisConfiguration() {
+    if (!System.getenv("ROBOT_TYPE").isEmpty() && System.getenv("ROBOT_TYPE").equals("PRACTICE"))
```

Comment:
1. Could be replaced with a ternary operator
2. Could change the env var name to PRACTICE_ROBOT and check if that var is present to simplify the operation.
3. Could also move this check onto line 68 for simplicity.

Reply:
Resolved 1 & 2. I believe that it is still beneficial to tell the robot apart via the getChassisConfiguration method now rather than moving the checking logic to setOffsets (especially for characterization constants, there MAY be some variance; hopefully I'm not talking out of my ass)